docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,109 @@
+the installation steps.
+
+## Interface Overview
+
+![interface overview of Teleport Connect](../../img/use-teleport/connect-ui-overview.png)
+
+The top bar of Teleport Connect consists of:
+
+- The profile selector (the top right) allowing you to switch between profiles on different Teleport
+  clusters as well as log in or out of the clusters.
+- The connection list (the top left) showing recent connections, allowing you to seamlessly switch
+  between them.
+- The command bar (in the middle) which launches any command you input in a new terminal tab.
+- The cluster selector (to the left of the command bar) shows up only if there are leaf clusters
+  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
+  which cluster the commands in the command bar are going to target.
+
+## Connecting to an SSH Server
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Look for the SSH server you want to connect to and click the Connect button to the right.
+3. Select or enter the login you wish to use and press `Enter`.
+4. A new tab will open with a shell session on the chosen server.
+
+Alternatively, you can type `tsh ssh` into the command bar and the autocompletion will guide you to
+connect to a server.
+
+## Connecting to a Database
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Select the Databases section.
+3. Look for the database server you wish to connect to and click the Connect button to the right.
+4. Select or enter the database user you with to use and press `Enter`.
+5. A new tab will open with a new connection established between your device and the database
+   server.
+
+This connection will remain active until you click the Close Connection button or close Teleport
+Connect. The port number will persist between app restarts – you can set up your favorite client
+without worrying about the port suddenly changing.
+
+### With a GUI Client
+
+To connect with a GUI client, follow the instructions in the database connection tab under the
+Connect with GUI section.
+
+### With a CLI Client
+
+The database connection tab shows the command that can be used to connect to the database. You can
+modify the database name of the connection and then click the Run button to open a new
+terminal tab with that command executed.
+
+## Connecting to Multiple Clusters
+
+After you add your first cluster, you can keep adding more by opening the profile selector in the
+top right and clicking at the *+ Add another cluster* button.
+
+At the moment Teleport Connect supports only one user per cluster. To log in as a different user,
+log out of the cluster first.
+
+## Restarting and Reconnecting
+
+Teleport Connect remembers the tabs that you had open and will ask you on restart if you want to
```

Comment:
Not sure if this change is needed, but maybe reword it around "the previous session" or something around the word session since the modal that pops up when you open says "start new session" or reopen. lets use the "session" wording somewhere. "Connect session" maybe?

Reply:
Yeah that makes sense, also that sentence was super long. I changed it to the following:

> Before closing, Teleport Connect will remember the tabs that you had open at the end of the previous
> session. Next time you open it, Connect will ask you if you want to reopen those tabs. If you agree,
> Connect will restore connections to all resources that were active before you closed the app.

That "If you agree" part sounds weird to me though, now that I think about it.